src/display/network.js

```diff
@@ -14,10 +14,11 @@
  */
 
 import {
-  assert, createPromiseCapability, globalScope, isInt, MissingPDFException,
+  assert, createPromiseCapability, globalScope, MissingPDFException,
   UnexpectedResponseException
 } from '../shared/util';
 import { setPDFNetworkStreamClass } from './api';
```

Comment:
I think there is some confusion at https://github.com/mozilla/pdf.js/pull/8712#discussion_r131554047. Done with new commit, Thanks.